client/src/components/elements/Humanize.js

```diff
@@ -0,0 +1,12 @@
+import PropTypes from 'prop-types';
+
+const Humanize = ({children}) => {
+  const [name] = (children || '').split('/').slice(-1);
```

Comment:
Ideally, think we should avoid reliance of format of API data to derive a display name.
If I'm reading correctly, this getting an identifier (e.g: cgc name) and removing hyphens/underscores, such that:
"abc-1" -> "abc 1"
or
"abc_123" -> "abc 123" ?
Think it would be ok to leave the identifiers "as is" if this is the case...

Reply:
Oh, this is actually for displaying keywords, say to turn "provenance/merged-from" into "merged from".

Reply:
better to have a static mapping from keyword to display name?

Reply:
Yup, this is the mainly for the fall through cases anyway. I didn't think too much about this when I implemented this. Will fix if this causes problems later on.